docs/guidance_for_upgraders.md

```diff
@@ -0,0 +1,74 @@
+# Guidance for Operator Developer
+
+After you change either the operator signature in a BC breaking way or the semantics of the operator, you will need to write an “upgrader” to make the change non-BC breaking iff they are used in TorchScript or mobile. In general, you can know your operator is BC breaking, if it fails `test/forward_backward_compatibility/check_forward_backward_compatibility.py `
```

Comment:
Break this into multiple lines.

s/signature/schema/

s/your operator is BC breaking/your change to operator is BC-breaking

Reply:
Resolve